README.md

```diff
@@ -55,20 +55,18 @@ await client.workspaces.deleteWorkspace(workspace);
 
 ### Schema-generated Client
 
-First of all, add an npm script to your package.json file to invoke `xata-codegen`. You can customize the location of the schema file and the generated output file. For example:
+To use the schema-generated client, you need to run the code generator utility that comes with our CLI.
 
-```json
-{
-  "scripts": {
-    "xata-codegen": "xata-codegen xata -o src/xata.ts"
-  }
-}
+To run it (and assuming you have configured the project with `xata init`):
+
+```bash
+xata codegen
 ```
 
-Run it (this is assuming your Xata directory is in `./xata` and it was created by the `xata` cli application)
+If you want to override the output file, you can pass the ``--out`` flag with the path to the output file.
```

Comment:
This flag was removed, but I'm planning to re-add it with #328

Reply:
When I just used `xata init` it added 

```  
  "codegen": {
    "output": "./db/xata.ts"
  }
```

to my `.xatarc`.

Would/should I pass the `--out` argument?

Reply:
Not needed, it is optional if you want to override it. Maybe we can remove it from the docs, they can figure it out with `--help`